client/coral-admin/src/actions/comments.js

```diff
@@ -16,3 +15,23 @@ export const flagComment = id => (dispatch, getState) => {
 export const createComment = (name, body) => dispatch => {
   dispatch({type: 'COMMENT_CREATE', name, body});
 };
+
+// Dialog Actions
+export const showBanUserDialog = (userId, userName, commentId) => {
```

Comment:
Looks 💯 

This:
```js
export const showBanUserDialog = (userId, userName, commentId) => {
  return dispatch => {
    dispatch({type: 'SHOW_BANUSER_DIALOG', userId, userName, commentId});
  };
};
```
is the same as doing: 

```js
export const showBanUserDialog = (userId, userName, commentId) => dispatch => ({
  dispatch({type: 'SHOW_BANUSER_DIALOG', userId, userName, commentId});
})
```

The () is implicit return `() => (something)`